EIPS/eip-2628.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: eip-2628
+title: Header in StatusMessage
```

Comment:
I think this is going to be something like `eth/66: Header in StatusMessage` and the description then proposes a change to the existing protocol (eth/65?).

See https://eips.ethereum.org/EIPS/eip-2464 as an example.

@fjl @karalabe can you guys give some directions here?

Reply:
There's already an eth/66 from @cburgdorf

Reply:
Just linking to the [`eth/ 66` EIP](https://github.com/ethereum/EIPs/pull/2481/files)

Reply:
https://eips.ethereum.org/EIPS/eip-2481 aka "eth/66" was merged, so I guess this is "eth/67" then?